### PR TITLE
Feedback changes 

### PR DESCRIPTION
**Pull request by nitinprojects**
- Updated notes description
- Added util to convert string to camel case

### CODE REVIEW

File: app/javascript/src/utils/index.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 
+const toCamelCase = str =>
+  str
+    .split(" ")
+    .map((word, i) => {
+      if (i === 0) return word.toLowerCase();
+
+      return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+    })
+    .join("");
```
Comment (yedhink) [inline app/javascript/src/utils/index.js:11]: So what will be the output if the label is `Ux-Designer` or `UX_designer` or `UX    designer` etc? My expectation is that in all those cases the output should be `uxDesigner`.

**Review by yedhink — COMMENTED** [review]
@nitinprojects

File: app/javascript/src/utils/index.js (hunk 1 above)
Comment (nitinprojects) [reply app/javascript/src/utils/index.js:11]: I just thought of spaces for now, it will not work for the other two cases.

Comment (nitinprojects) [reply app/javascript/src/utils/index.js:11]: I have handled the other two cases here [#20](https://github.com/nitinprojects/wheel/pull/20/files)

**Pull request merged**